### ISSUE TITLE
关于代码的问题

### ISSUE THREAD

**Issue by Hunter-P** [penghu-cs/DSCMR#6]
您好！我想请问：
在train_model.py中，计算loss的输入里面，为什么要输入两个一样的labels，这样labels的loss不就没有意义了吗？
loss = calc_loss(view1_feature, view2_feature, view1_predict,
                                     view2_predict, labels, labels, alpha, beta)

**Comment by timtian12** [comment]
我也想问这个问题，同时这里得label信息指的是图片得类别吗？我看代码中最后输出得是一个10个节点得全连接，没有经过sigmod变换，直接进行得输出吗？还有theta11、theta22两个imge和text之间得相似度，这里计算得是同一个图片和文本吗？  期待回复
